.github/workflows/continuous-integration.yml

```diff
@@ -248,8 +251,8 @@
           with:
             sauce-username: ${{ secrets.SAUCE_USERNAME }}
             sauce-access-key: ${{ secrets.SAUCE_ACCESS_KEY }}
-            aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY }}
-            aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+            aws-role-to-assume: ${{ secrets.AWS_ROLE_TO_ASSUME_SDK_DEV }}
+            aws-role-session-name: ${{ env.TEST_TYPE }}
```

Comment:
What is the test type?

Reply:
https://github.com/aws/amazon-chime-sdk-js/blob/33cbc57c983417a00a1c78f0b2a6aca6c182166b/.github/workflows/continuous-integration.yml#L12